website/docs/API.md

```diff
@@ -563,7 +563,7 @@ It is also recommended to have a [single assertion per each `waitFor`](https://k
 ### Using Jest fake timers
 
 :::caution
-When using modern fake timers (the default for `Jest` >= 27), `waitFor` won't work (it will always timeout even if `expectation()` doesn't throw) unless you use the custom [@testing-library/react-native preset](https://github.com/callstack/react-native-testing-library#custom-jest-preset). 
+When using modern fake timers (the default for `Jest` >= 27), `waitFor` won't work (it will always timeout even if `expectation()` doesn't throw) unless you use the custom [@testing-library/react-native preset](https://github.com/callstack/react-native-testing-library#custom-jest-preset) or you use a version of React Native >= 0.71. 
```

Comment:
Can we flip the order of conditions explained in the docs? I would rather explain it in a way that:
1. if you use React Native earlier that 0.71
2. and you use modern Jest fake timers which are default for Jest 27+
3. then you should use custom preset or `waitFor` + fake timers will not work

Reply:
Agreed, I changed it and also the title